### DISCUSSION TITLE
Headers on admin

### DISCUSSION THREAD

**Discussion by kefayati** [dropwizard/dropwizard-web#383] [Ideas]
I needed to set headers on the admin site, which is typically spun up on a different port and different servlets. Checking the code, it seems using `environment.servlets()` is hardcoded and there is no option to choose `environment.admin()` and add the header filter there. Is there a reason for that? It is a relatively simple fix and I am happy to do a PR to add that option but I was wondering if there is something I am missing.

**Comment by joschi** [comment]
@kefayati Sorry for the late reply.

This feature has been implemented in the latest releases of dropwizard-web:

- https://github.com/dropwizard/dropwizard-web/discussions/502
- https://github.com/dropwizard/dropwizard-web/discussions/503
- https://github.com/dropwizard/dropwizard-web/discussions/504

```java
bootstrap.addBundle(new WebBundle<>() {
    @Override
    public WebConfiguration getWebConfiguration(final ExampleConfiguration configuration) {
        return configuration.getWebConfiguration();
    }

    // Optional: Override Servlet environment to apply the configuration to the admin servlets
    @Override
    protected ServletEnvironment getServletEnvironment(Environment environment) {
        return environment.admin();
    }
});
```
